samtools/1.18/Dockerfile

```diff
@@ -0,0 +1,91 @@
+ARG SAMTOOLS_VER="1.18"
+
+FROM ubuntu:jammy as builder
+
+ARG SAMTOOLS_VER
+
+# install dependencies required for compiling samtools
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    libncurses5-dev \
+    libbz2-dev \
+    liblzma-dev \
+    libcurl4-gnutls-dev \
+    zlib1g-dev \
+    libssl-dev \
+    gcc \
+    wget \
+    make \
+    perl \
+    bzip2 \
+    gnuplot \
+    ca-certificates
+
+# download, compile, and install samtools
+RUN wget https://github.com/samtools/samtools/releases/download/${SAMTOOLS_VER}/samtools-${SAMTOOLS_VER}.tar.bz2 && \
+    tar -xjf samtools-${SAMTOOLS_VER}.tar.bz2 && \
+    cd samtools-${SAMTOOLS_VER} && \
+    ./configure && \
+    make && \
+    make install
+
+### start of app stage ###
+FROM ubuntu:jammy as app
+
+ARG SAMTOOLS_VER
+
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="samtools"
+LABEL software.version="${SAMTOOLS_VER}"
+LABEL description="Tools (written in C using htslib) for manipulating next-generation sequencing data"
+LABEL website="https://github.com/samtools/samtools"
+LABEL license="https://github.com/samtools/samtools/blob/develop/LICENSE"
+LABEL maintainer="Shelby Bennett"
+LABEL maintainer.email="shelby.bennett@dgs.virginia.gov"
+LABEL maintainer2="Curtis Kapsak"
+LABEL maintainer2.email="kapsakcj@gmail.com"
+LABEL maintainer3="Erin Young"
+LABEL maintainer3.email="eriny@utah.gov"
+LABEL maintainer4="Kutluhan Incekara"
+LABEL maintainer4.email="kutluhan.incekara@ct.gov"
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# install dependencies required for running samtools
+# 'gnuplot' required for plot-ampliconstats
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    perl \
+    zlib1g \
+    libncurses5 \
+    bzip2 \
+    liblzma-dev \
+    libcurl4-gnutls-dev \
+    gnuplot \
+    && apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# copy in samtools executables from builder stage
+COPY --from=builder /usr/local/bin/* /usr/local/bin/
+
+ENV LC_ALL=C
+
+# final working directory is /data
+WORKDIR /data
+
+# default command is to pull up help options
+CMD ["samtools", "--help"]
+
+### start of test stage ###
+FROM app as test
+
+# install wget for downloading test files
+RUN apt-get update && apt-get install --no-install-recommends -y wget \
+ ca-certificates 
```

Comment:
ca-certificates and procps need to be in the app layer. It's required for nextflow (maybe newer versions don't need it, though, so I'm going to be investigation this for a moment).